Fix linting

This repository is misaligned with other repositories due to not being linted and a different config.

The following should be set in the eslint config, `"es6": true`, and linting issues addressed.

@abhayks1 noted that it may be required to make some changes to the code after enabling `es6` in the config of eslint.

```
Abhays-MacBook-Pro:brandedtoken-contracts abhay$ ./node_modules/eslint/bin/eslint.js --config .eslintrc .

/Users/abhay/Documents/projects/brandedtoken-contracts/dist/index.js
  21:27  error  Unable to resolve path to module './contracts.json'  import/no-unresolved

/Users/abhay/Documents/projects/brandedtoken-contracts/migrations/1_initial_migration.js
  3:18  warning  Unexpected unnamed function                   func-names
  4:1   error    Expected indentation of 2 spaces but found 4  indent

/Users/abhay/Documents/projects/brandedtoken-contracts/test/utility_branded_token/increase_supply.js
  31:7  error  'worker' is assigned a value but never used  no-unused-vars

/Users/abhay/Documents/projects/brandedtoken-contracts/truffle.js
   2:1  error  Expected indentation of 2 spaces but found 4   indent
   3:1  error  Expected indentation of 4 spaces but found 8   indent
   4:1  error  Expected indentation of 6 spaces but found 12  indent
   5:1  error  Expected indentation of 6 spaces but found 12  indent
   6:1  error  Expected indentation of 6 spaces but found 12  indent
   7:1  error  Expected indentation of 6 spaces but found 12  indent
   8:1  error  Expected indentation of 6 spaces but found 12  indent
   9:1  error  Expected indentation of 4 spaces but found 8   indent
  10:1  error  Expected indentation of 2 spaces but found 4   indent
  11:1  error  Expected indentation of 2 spaces but found 4   indent
  12:1  error  Expected indentation of 4 spaces but found 8   indent
  13:1  error  Expected indentation of 4 spaces but found 8   indent
  14:1  error  Expected indentation of 4 spaces but found 8   indent
  15:1  error  Expected indentation of 4 spaces but found 8   indent
  16:1  error  Expected indentation of 4 spaces but found 8   indent
  17:1  error  Expected indentation of 2 spaces but found 4   indent
  18:1  error  Expected indentation of 2 spaces but found 4   indent
  19:1  error  Expected indentation of 4 spaces but found 8   indent
  20:1  error  Expected indentation of 6 spaces but found 12  indent
  21:1  error  Expected indentation of 6 spaces but found 12  indent
  22:1  error  Expected indentation of 6 spaces but found 12  indent
  23:1  error  Expected indentation of 6 spaces but found 12  indent
  24:1  error  Expected indentation of 6 spaces but found 12  indent
  25:1  error  Expected indentation of 4 spaces but found 8   indent
  26:1  error  Expected indentation of 2 spaces but found 4   indent

✖ 29 problems (28 errors, 1 warning)
  26 errors and 0 warnings potentially fixable with the `--fix` option.
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -11,7 +11,7 @@
     "env": {
         "mocha": true,
         "node": true,
-        "es6": false
+        "es6": true
     },
     "globals": {
         "artifacts": false,

--- a/.travis.yml
+++ b/.travis.yml
@@ -23,6 +23,7 @@ install:
 before_script:
   - nohup sh tools/runGanacheCli.sh </dev/null >/dev/null 2>&1 &
 script:
+  - npm run lint
   - npm run test
   - npm run compile-all
   - npm run build-package

--- a/migrations/1_initial_migration.js
+++ b/migrations/1_initial_migration.js
@@ -1,5 +1,5 @@
 const Migrations = artifacts.require('./Migrations.sol');
 
-module.exports = function (deployer) {
-    deployer.deploy(Migrations);
+module.exports = (deployer) => {
+  deployer.deploy(Migrations);
 };

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
     "compile-all": "truffle compile --all",
     "test": "truffle test",
     "prepack": "npm run compile-all && npm run build-package",
-    "lint": "eslint {migrations,test,tools} -c .eslintrc --ext js"
+    "lint": "eslint migrations/*.js test/*/*.js tools/*.js -c .eslintrc"
   },
   "main": "dist/index.js",
   "files": [

--- a/package.json
+++ b/package.json
@@ -25,7 +25,8 @@
     "build-package": "node tools/build_package.js",
     "compile-all": "truffle compile --all",
     "test": "truffle test",
-    "prepack": "npm run compile-all && npm run build-package"
+    "prepack": "npm run compile-all && npm run build-package",
+    "lint": "eslint {migrations,test,tools} -c .eslintrc --ext js"
   },
   "main": "dist/index.js",
   "files": [

--- a/test/utility_branded_token/increase_supply.js
+++ b/test/utility_branded_token/increase_supply.js
@@ -28,7 +28,6 @@ contract('UtilityBrandedToken::increaseSupply', async (accounts) => {
   let tokenHolder3;
   let accountProvider;
   let coGateway;
-  let worker;
 
   const amount = 10;
   const tokenHolder1Balance = 100;
@@ -46,7 +45,6 @@ contract('UtilityBrandedToken::increaseSupply', async (accounts) => {
 
     ({
       testUtilityBrandedToken,
-      worker,
     } = await UtilityBrandedTokenUtils.setupUtilityBrandedToken(
       accountProvider, internalActors,
     ));

